src/components/tasks/card/index.tsx

```diff
@@ -31,6 +31,7 @@ import { useEditMode } from '@/hooks/useEditMode';
 import { useGetUsersByUsernameQuery } from '@/app/services/usersApi';
 import { ConditionalLinkWrapper } from './ConditionalLinkWrapper';
 import { isNewCardDesignEnabled } from '@/constants/FeatureFlags';
+import { isTaskDetailsPageLinkEnabled } from '@/constants/FeatureFlags';
```

Comment:
In case of disabling the feature, it will be `false` and in the rest of the cases it will be `true`

Reply:
but this will always be `true` as you have imported from constant. there isn't any code which will be changing this based on any condition.

Reply:
It will be `TRUE` always, but we can add code to **make it false in the future**, whenever we want to disable this feature and set this to TRUE for making this feature live.